cfme/tests/v2v/test_v2v_smoke_tests.py

```diff
@@ -76,3 +80,49 @@ def _cleanup():
     # validate MAC address matches between source and target VMs
     migrated_vm = get_migrated_vm_obj(src_vm_obj, provider)
     assert src_vm_obj.mac_address == migrated_vm.mac_address
+
+
+@pytest.mark.parametrize(
+    "v2v_provider_setup, mapping_data_multiple_vm_obj_single_datastore",
+    [["SSH", ["nfs", "nfs", [rhel7_minimal]]]],
+    indirect=True,
+)
+def test_single_vm_migration_with_ssh(
```

Comment:
You can also remove entire `test_migration_plan` by adding `[["SSH", ["nfs", "nfs", [rhel7_minimal]]], ["VDDK", ["nfs", "nfs", [rhel7_minimal]]]]` here. This should cover RHV + VDDK/SSH and OSP + VDDK/SSH (just need to test OSP by executing).